french/maps/outback_01.rpy

```diff
@@ -1,146 +1,145 @@
-﻿# TODO: Translation updated at 2024-07-14 12:32
+﻿﻿# TODO: Translation updated at 2024-07-14 12:32
 
 # game/maps/outback_01.rpy:183
 translate french outback_talk_01_ab69ebed:
 
     # p surprise "Hmmm...{w=0.5} Puzzling."
-    p surprise ""
+    p surprise "Hmmm...{w=0.5} Déroutant."
 
 # game/maps/outback_01.rpy:184
 translate french outback_talk_01_29002587:
 
     # e surprise "What is?"
-    e surprise ""
+    e surprise "De quoi ?"
 
 # game/maps/outback_01.rpy:191
 translate french outback_talk_01_a290ff97:
 
     # p nulla "I was just thinking, Ruby buys each month enough supplies for herself and Maria...{w=0.5} Even without accounting for water, that's quite the sizeable amount."
-    p nulla ""
+    p nulla "J'étais juste en train de réfléchir, Ruby achète tous les mois assez de provisions pour elle-même et Maria...{w=0.5} Sans même prendre en compte de l'eau, cela représente une quantité assez importante."
 
 # game/maps/outback_01.rpy:192
 translate french outback_talk_01_945e63e6:
 
     # e nulla "Oh, I see.{w=0.3} She must have some kind of vehicle or transport to get them to point B, after being dropped off here."
-    e nulla ""
+    e nulla "Oh, je vois.{w=0.3} Elle doit avoir une sorte de véhicule de transport pour les emmener au point B, après être déposée ici."
 
 # game/maps/outback_01.rpy:194
 translate french outback_talk_01_6afa6b5c:
 
     # p nulla "That was my idea.{w=0.3} Yet, I don't see any tracks which would correspond in our vicinity..."
-    p nulla ""
+    p nulla "C'était mon idée.{w=0.3} Pourtant, je ne vois aucune trace qui pourrait correspondre dans notre voisinage."
```

Comment:
vicinity -> environs
"I don't see any tracks which would correspond in our vicinity..." -> "[...] correspondre à nos environs."